javascript/ImageClassification/index.html

```diff
@@ -11,8 +11,8 @@
   <meta charset="UTF-8">
   <title>Image Classification Example</title>
 
-  <script src="https://unpkg.com/ml5" type="text/javascript"></script>
-  
+  <!-- <script src="https://unpkg.com/ml5" type="text/javascript"></script> -->
+  <script src="http://localhost:8080/ml5.js"></script>
```

Comment:
Looks like some references to `localhost` instead of `unpkg` left here by accident.

Reply:
Yes, not until we update to the newest version. We should also switch to specify the version `https://unpkg.com/ml5@0.0.8/dist/ml5.min.js` since its more clear